src/DETHRACE/common/finteray.c

```diff
@@ -178,6 +178,7 @@ int DRSceneRayPick2D(br_actor* world, br_vector3* pPosition, br_vector3* pDir, d
     LOG_TRACE("(%p, %p, %p, %p)", world, pPosition, pDir, callback);
 
     BrMatrix34Inverse(&gPick_model_to_view__finteray, &world->t.t.mat);
+    LOG_WARN_ONCE("Missing material and model pointers to ActorRayPick2D");
```

Comment:
Whats the purpose of the warning here? These params are used as defaults and are always deliberately set to null

Reply:
I think this function is called with a material and model pointer, but I don't know what one and how it is used.